From d73f1f024a8f179e38ce15dc66938ee40325e5b3 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 3 Apr 2013 08:33:28 +0200
Subject: [PATCH 03/14] beaglebone: HACK: always return 1 for is_bone_lt

This is to make the eMMC flasher work, the EEPROM hasn't been populated yet

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 board/ti/am335x/board.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/board/ti/am335x/board.c b/board/ti/am335x/board.c
index b371376..c92e1e9 100644
--- a/board/ti/am335x/board.c
+++ b/board/ti/am335x/board.c
@@ -60,7 +60,8 @@ static inline int board_is_bone(void)
 
 static inline int board_is_bone_lt(void)
 {
-	return !strncmp(header.name, "A335BNLT", HDR_NAME_LEN);
+	//return !strncmp(header.name, "A335BNLT", HDR_NAME_LEN);
+	return 1;
 }
 
 static inline int board_is_evm_sk(void)
@@ -378,7 +379,7 @@ void s_init(void)
 	if (board_is_evm_sk())
 		config_ddr(303, MT41J128MJT125_IOCTRL_VALUE, &ddr3_data,
 			   &ddr3_cmd_ctrl_data, &ddr3_emif_reg_data, 0);
-	else if (board_is_bone_lt())
+	else if (board_is_bone_lt() && !board_is_bone())
 		config_ddr(400, MT41K256M16HA125E_IOCTRL_VALUE,
 			   &ddr3_beagleblack_data,
 			   &ddr3_beagleblack_cmd_ctrl_data,
-- 
1.8.1.4

